packages/cardpay-reward-programs/tests/rules/test_dummy_rule.py

```diff
@@ -0,0 +1,194 @@
+import duckdb
+import pandas as pd
+from cardpay_reward_programs.rules import dummy_rule
+
+
+def create_rule(
+    monkeypatch,
+    fake_data_spend_acc,
+    fake_data_safe_own,
+    core_config_overrides={},
+    user_config_overrides={},
+):
+
+    core_config = {
+        "payment_cycle_length": 32768,
+        "start_block": 0,
+        "end_block": 10000,
+        "subgraph_config_locations": {
+            "prepaid_card_payment": "s3://cardpay-staging-partitioned-graph-data/data/prepaid_card_payments/0.0.3/",
+            # "spend_accumulation": "s3://tall-data-dev/paulin/spend_accumulation/0.0.1/",
+            "safe_owner": "s3://tall-data-dev/paulin/safe_owner/0.0.1/",
+        },
```

Comment:
I don't think this data is available in the repo. Hence, we r probably making real s3 calls rather than mocking

Reply:
yeah I'm going to pull the dummy rule work out, we can bring it back in for examples at some point but I think there's more cleanup required than I want to do for getting staking out.